### ISSUE TITLE
Move already seen banners on top of subject

### ISSUE THREAD

**Issue by srallen** [zooniverse/front-end-monorepo#932]
**Package** lib-classifier

**Feature or Issue Description**
It's impossible to remove the column data annotation with the banners:

![Image from iOS](https://user-images.githubusercontent.com/5016731/59776567-aeecdc00-9278-11e9-8cf7-8dc7a96434ae.png)

**Comment by rogerhutchings** [comment]
The other option is to move them vertically above the the subject viewer, rather than have them as an overlay.

Less code, less chance of forgetting you're on a completed workflow / any other special circumstance where your classification may be less useful.

**Comment by srallen** [comment]
True, but I'll leave that up to @beckyrother. I think the overlay design was to try to save vertical space.

**Comment by beckyrother** [comment]
Thanks for tagging me @srallen. This placement was to save vertical space and make sure all the banners were in a consistent location. I'd be open to seeing it above the image, though, especially seeing them on top of data like this – when I designed this, there were only images and video.

**Comment by srallen** [comment]
@beckyrother I don't have a story for you to look at, but I rearranged the position in a hacky way using Chrome dev tools. Here are some screenshots (pretend the image toolbar is the right height!):

<img width="1384" alt="Screen Shot 2019-10-22 at 4 40 46 PM" src="https://user-images.githubusercontent.com/5016731/67336335-c53acb00-f4ea-11e9-99d4-0966b7f846c7.png">
<img width="1394" alt="Screen Shot 2019-10-22 at 4 40 57 PM" src="https://user-images.githubusercontent.com/5016731/67336337-c835bb80-f4ea-11e9-9d9d-72fbbaeb7ae6.png">

I also changed the red to have full opacity. It was at 0.8 when it was the overlay.
